### PR TITLE
Update issue templates with new supported values

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
GitHub now supports new values on issue templates, so here they are.  

  

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Release_Process.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ---
 name: Release Process
 about: Maintainers Only - Release Process
+title: "[RELEASE] Release Issue"
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Release_Process.md]: Can we change the title to `Release Process for v6.x.x`

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 name: Security Issue
 about: Security Issue
+title: "[SECURITY] Security Issue"
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Security_Issue_Report.md]:
```suggestion
title: "[SECURITY] !!! READ TEMPLATE COMPLETELY FIRST!!!"
```

File: .github/ISSUE_TEMPLATE/Release_Process.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
+labels: maintainers
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Release_Process.md]:
```suggestion
labels: Issue-Meta
```

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 ---
 name: Feature Request/Idea 🚀
 about: Suggest a new feature or improvement (this does not mean you have to implement it)
+title: "[FEATURE] Feature Request"
+labels: enhancement
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Feature_Request.md]:
```suggestion
labels: Issue-Enhancement
```

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 ---
 name: Bug report 🐛
 about: Report errors or unexpected behavior 🤔
+title: "[BUG] My bug report"
+labels: bug
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]:
```suggestion
labels: Issue-Bug
```

**Comment by RDIL** [comment]
Suggestions accepted.

File: .github/ISSUE_TEMPLATE/Support_Question.md [hunk 6]
```diff
@@ -1,9 +1,13 @@
 ---
 name: Support Question ❓
-about: If you have a question, you can try asking in the PowerShell Slack channel first. If you need official support, refer to the [PowerShell Support Lifecycle](http://aka.ms/pslifecycle)
+about: If you have a question, you can try asking in the PowerShell Slack channel first. If you need official support, refer to the [PowerShell Support Lifecycle](https://aka.ms/pslifecycle)
+title: "[SUPPORT] Support Question"
+labels: support
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Support_Question.md]:
```suggestion
labels: Issue-Question
```

File: .github/ISSUE_TEMPLATE/Windows_PowerShell.md [hunk 7]
```diff
@@ -1,6 +1,8 @@
 ---
 name: Windows PowerShell
 about: Windows PowerShell issues/suggestions need to be reported to [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
+labels: windows
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Windows_PowerShell.md]:
```suggestion
labels: Issue-Question
```

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md [hunk 8]
```diff
@@ -1,6 +1,9 @@
...
+labels: secuirty
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Security_Issue_Report.md]:
```suggestion
labels: Issue-Bug
```

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md [hunk 9]
```diff
@@ -1,6 +1,9 @@
...
+assignees: ''
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Security_Issue_Report.md]:
```suggestion
assignees: 'TravisEz13'
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
Now we have many issues with bug label what doesn't look right :-( I think the label should be assigned only for confirmed bugs.

@TravisEz13 @SteveL-MSFT Thoughts?

**Comment by SteveL-MSFT** [comment]
@iSazonov I agree, the original intent of the `Issue - Bug` label is when it's a identified bug, not a possible bug.  Perhaps we should add a `Issue - Needs Triage` label that gets put automatically, then after someone has taken a look, it can be bug, discussion, etc...?

**Comment by iSazonov** [comment]
We could reuse "discussion" label - in any case,sooner or later, issues with "discussion" should be triaged to bug, enhancement , up-to-grabs or closed by-design or wont-fix.
Also we could don't initially label issues with bug template at all and make this manually after first review or after initial discussion.

**Comment by SteveL-MSFT** [comment]
@iSazonov that might work for now and we can revisit later anyways, do you want to submit a PR?

**Pull request merged**